usaspending_api/settings.py

```diff
@@ -93,17 +93,27 @@
 if not STATE_DATA_BUCKET:
     STATE_DATA_BUCKET = os.environ.get("STATE_DATA_BUCKET")
 
-DATA_DICTIONARY_DOWNLOAD_URL = "https://files{}.usaspending.gov/docs/Data_Dictionary_Crosswalk.xlsx".format(
-    "-nonprod" if DOWNLOAD_ENV != "production" else ""
-)
+# Download URLs
+FILES_SERVER_BASE_URL = ""
+SERVER_BASE_URL = ""
+
+if not FILES_SERVER_BASE_URL:
```

Comment:
Does this allow us to not have to add any config changes to `usaspending-api` `master` branch (DAOI infra)? i.e. we just leave out the setting in DAOI, and it falls back to a good default? 
But in DTI, we set it to the "new" URL and it will pick it up?

Reply:
That is correct. I tested in DAOI Sandbox to make sure this worked as expected. My comment below is to remind that after the migration we can remove this default value.